source/ceylon/buffer/codec/IncrementalCodec.ceylon

```diff
@@ -23,8 +22,10 @@ shared class ChunkConvert<ToMutable, FromImmutable, ToSingle, FromSingle>(conver
     shared Boolean done => remainder.empty;
```

Comment:
But as I said, the remainder is only half the story. There may be stuff in the PieceConverter as well. So there will have to be some other API if you want to empty everything out without feeding the `ChunkConvert` a `finished` and draining it with a buffer that has free space (read until 0 is returned).

If you disagree, can you provide a code example of how `done` would be properly used?